README.md

```diff
@@ -27,17 +48,21 @@ https://timowest12.github.io/Awesome-books/
 
 ![Shoira's GitHub stats](https://github-readme-stats.vercel.app/api?username=shoirata&count_private=true&theme=dark&show_icons=true)
 
-👤 **Timo Wester**
+👤 **Daniel Shdeed**
 
-- GitHub: [@githubhandle](https://github.com/Timowest12)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/timo-wester-6a0282a7/)
+- GitHub: [@githubhandle](https://github.com/Danieldotcomcoder))
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/daniel-shdeed-832b03115/)
 
-![Timo's GitHub stats](https://github-readme-stats.vercel.app/api?username=Timowest12&count_private=true&theme=dark&show_icons=true)
+![Daniel's GitHub stats](https://github-readme-stats.vercel.app/api?username=danieldotcomcoder&count_private=true&theme=dark&show_icons=true)
 
 ## Acknowledgments
 
 - Microverse
 
 ## 📝 License
 
-This project is [MIT](./MIT) licensed.
+This project is [MIT](<.(https://choosealicense.com/licenses/mit/)>) licensed.
```

Comment:
- [ ] [OPTIONAL] If you want to make your project look even more professional you might want to consider adding the MIT.md file to your repo and adding a link to it in your Readme.md file like is done in [the template provided by Microverse](https://github.com/microverseinc/readme-template).